### ISSUE TITLE
The animation configuration is wrong/missing

### ISSUE THREAD

**Issue by Joelius300** [mariusmuntean/ChartJs.Blazor#86]
### Describe the bug
This bug was brought up in a [StackOverflow question](https://stackoverflow.com/questions/60834550/blazor-reduce-elimination-of-animation-speed-of-a-chart/).

Not all options have an animation property even though all options appear to support it [source](https://www.chartjs.org/docs/latest/configuration/animations.html). Furthermore `ArcAnimation` should be a sub-class of `Animation`. Also the easing property is missing in `Animation`. For that we need a new string enum which we can also use in #57 once we get to it.  
The callbacks are also missing but they can only be done after #70 is merged.

### Which charts does this bug apply to?
All of them even though some avoid the trouble by having their own animation property.

### How to fix
- Subclass `ArcAnimation` from `Animation`
- Add `Animation` property to `BaseConfigOptions`
- Change type to `ArcAnimation` or explicitly mention `ArcAnimation` for the `Animation` property of `PolarAreaOptions` and `PieOptions`.
- Add `Easing` string enum
- Add `Easing` property to `Animation`
- Think about the callbacks, maybe open another issue

### Additional info
Most of the info we need can be found under the [animation section](https://www.chartjs.org/docs/latest/configuration/animations.html) in the chart.js-docs.

`ArcAnimation` is used for the pie, doughnut and polar area chart but I don't know if it's possible to make it of that type in those options. Maybe just write it in the summary if we can't enforce the type.

An attempt at fixing this issue was made by @ItsMeDan11 in #74 (thank you!) but unfortunately it doesn't consider all the changes and just fixes it for `BarOptions`. I will probably take that PR as a reference and create a new one in order to fix this but I don't know yet.
